src/styles/mixins/molecules.scss

```diff
@@ -20,9 +20,9 @@
  * annotation (thread) cards. Will vertically-space its children. Adds a
  * hover/active intensified box shadow and accounts for "theme-clean" affordances.
  */
-@mixin card {
+@mixin card($rhythm: var.$layout-space) {
```

Comment:
The updates to this module's mixins were because the `panel--compact` mixin was initially designed for the specific "design case" of the share-single-annotation small panel with the text-input field in it, instead of the more "generic" header-and-text layout that is used in the new UI path (not shareable). This fixes it such that the tighter padding and rhythm in the variant that has the input field is the _exception_ rather than the _rule_.